CONTRIBUTING.md

```diff
@@ -20,18 +20,18 @@ Example to run a single test case:
 
 ## Adding/updating commands
 
-First, update [commands.yml](internal/commandsgen/commands.yml) following the rules in that file. Then to regenerate the
-[commands.gen.go](internal/commands.gen.go) file from code, simply run:
+First, update [commands.yaml](internal/temporalcli/commands.yaml) following the rules in that file. Then to regenerate the
+[commands.gen.go](internal/temporalcli/commands.gen.go) file from code, simply run:
 
-    go run ./internal/cmd/gen-commands
+    make gen
```

Comment:
We support contributors on all of our supported platforms and therefore contributing guide should work across all platforms we support IMO